bot/cogs/fibonacci.py

```diff
@@ -0,0 +1,33 @@
+from discord.ext.commands import Bot, Cog, command
+
+
+def fibo_send(n):
+    a, b = 0, 1
+    sequence = []
+    while a < n:
+        sequence.append(a)
+        a, b = b, a + b
+    return "".join(str(x) + " " for x in sequence)
+
+
+class Fibonacci(Cog):
+    """A simple command to create a fibonacci number from input."""
+
+    @command(brief="Calculates the fibonacci sequence given by an input")
+    async def fib(self, ctx, num):
+
+        if len(num) < 20:
```

Comment:
I added a length limit to 1. Save my server's processor and 2. Prevent a bad request error (2000 character limit).